static/static_ip.sh

```diff
@@ -219,12 +221,13 @@ network:
          addresses: [$ADDRESS/24] # client IP address
          gateway4: $GATEWAY # gateway address
          nameservers:
-           addresses: [$DNS1,$DNS2] #name servers
+           addresses: [$DNSs] #name servers
```

Comment:
Why this change?

Reply:
Having the 1 DNSs variable allows me to set either [1.1.1.1,2.2.2.2] or [1.1.1.1] in the field with the comma or not based on if I only have 1 DNS server set or 2.

Reply:
:+1: